src/pages/docs/version-control/converting/index.md

```diff
@@ -41,12 +41,15 @@ Finally, add your default branch name in Branch Settings and click **Configure**
 ![configuring version control](/docs/projects/version-control/converting/configure-version-control.png)
 :::
 
-Your project is now configured with Version Control. You can see this change reflected on the left navigation of the page, where you can change branches. You can also confirm this in your Git repository. The `.octopus` directory will now be created, and it should contain the following files:
+Your project is now configured with Version Control. You can see this change reflected on the left navigation of the page, where you can change branches. You can also confirm this in your Git repository. The `.octopus` directory will now be created, and it should contain the following files and folders:
 
 - _deployment_process.ocl_
 - _deployment_settings.ocl_
 - _variables.ocl_
 - _schema_version.ocl_
+- _runbooks/_
+
+The runbooks/ directory will contain runbook-name.ocl files for any published runbooks. 
```

Comment:
We prob want to explain that any draft runbooks will get put in that subfolder at conversion time (the name of the folder I can't remember) and give some guidance on if people want consumers/triggers to use these they'll need to move them back to the main `runbooks` folder on the default branch?